api/sensor/samples/test_set.js

```diff
@@ -0,0 +1,20 @@
+console.log('Code start!');
```

Comment:
Why did you put your sample application's code into ```api/sensor/samples``` directory?
There has already been two sample applications in ```apps/sample```.

As I think, you have to move this code to ```apps/sample```.